index.js

```diff
@@ -2,328 +2,417 @@
  * Copyright (c) Microsoft Corporation. All rights reserved.
  * Licensed under the MIT License. See License.txt in the project root for
  * license information.
- * 
+ *
  * This file uses an outdated library.  Please see the readme to find the latest version.
  */
-'use strict';
+("use strict");
 
-const util = require('util');
-const msRestAzure = require('ms-rest-azure');
-const ComputeManagementClient = require('azure-arm-compute');
-const NetworkManagementClient = require('azure-arm-network');
-const ResourceManagementClient = require('azure-arm-resource').ResourceManagementClient;
-const KeyVaultManagementClient = require('azure-arm-keyvault');
+const util = require("util");
+const { ClientSecretCredential } = require("@azure/identity");
```

Comment:
Can you change this ClientSecretCredential to DefaultAzureCredential ? Because the service team is asking us to use DefaultAzureCredential in the sample. Thank you